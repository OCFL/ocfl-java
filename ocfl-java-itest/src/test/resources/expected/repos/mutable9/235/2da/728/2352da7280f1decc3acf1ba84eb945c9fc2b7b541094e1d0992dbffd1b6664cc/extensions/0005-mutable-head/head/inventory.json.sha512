b7f5acb2efd5c120e094a40b2b5342eb4243a2a6a9e572d458ac30a0d178a20f04f8f94ab742a82530fa1dd6b86eb9cfa4c8c6e02d86c14d7641b94b98378ea0  inventory.json
