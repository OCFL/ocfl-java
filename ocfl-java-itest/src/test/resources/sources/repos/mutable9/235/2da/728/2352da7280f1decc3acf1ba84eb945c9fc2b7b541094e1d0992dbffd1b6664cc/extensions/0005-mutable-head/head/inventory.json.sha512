7feec010ef7b3dffc52d05724aa3612939225697b4b864f298bd2a93b3f85a1bc05859476a3d276c6667b7d373cfffb8c5f106aa60e983bc67518712e405ec67  inventory.json
