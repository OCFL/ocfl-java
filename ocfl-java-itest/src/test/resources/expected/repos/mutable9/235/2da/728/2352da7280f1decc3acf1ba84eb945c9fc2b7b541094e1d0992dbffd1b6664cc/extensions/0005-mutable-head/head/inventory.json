{
  "id" : "o1",
  "type" : "https://ocfl.io/1.1/spec/#inventory",
  "digestAlgorithm" : "sha512",
  "head" : "v2",
  "contentDirectory" : "content",
  "fixity" : { },
  "manifest" : {
    "3eb3222d4b5629a27dede47a90c7665fcaff1305c2c6c645465edc8278768032ca223f9193f3e9d6bb7a8b950ed284030d256ce1ee33e1de7e9f3e2cb1a834a0" : [ "extensions/0005-mutable-head/head/content/r12/file4" ],
    "4cf0ff5673ec65d9900df95502ed92b2605fc602ca20b6901652c7561b302668026095813af6adb0e663bdcdbe1f276d18bf0de254992a78573ad6574e7ae1f6" : [ "v1/content/file2" ],
    "96a26e7629b55187f9ba3edc4acc940495d582093b8a88cb1f0303cf3399fe6b1f5283d76dfd561fc401a0cdf878c5aad9f2d6e7e2d9ceee678757bb5d95c39e" : [ "v1/content/file1" ],
    "b10ff867df18165a0e100d99cd3d27f845f7ef9ad84eeb627a53aabaea04805940c3693154b8a32541a31887dda9fb1e667e93307473b1c581021714768bd032" : [ "extensions/0005-mutable-head/head/content/r1/dir1/file3" ]
  },
  "versions" : {
    "v1" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "commit message",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "4cf0ff5673ec65d9900df95502ed92b2605fc602ca20b6901652c7561b302668026095813af6adb0e663bdcdbe1f276d18bf0de254992a78573ad6574e7ae1f6" : [ "file2" ],
        "96a26e7629b55187f9ba3edc4acc940495d582093b8a88cb1f0303cf3399fe6b1f5283d76dfd561fc401a0cdf878c5aad9f2d6e7e2d9ceee678757bb5d95c39e" : [ "file1" ]
      }
    },
    "v2" : {
      "created" : "2019-08-05T15:57:53Z",
      "message" : "update",
      "user" : {
        "name" : "Peter",
        "address" : "peter@example.com"
      },
      "state" : {
        "3eb3222d4b5629a27dede47a90c7665fcaff1305c2c6c645465edc8278768032ca223f9193f3e9d6bb7a8b950ed284030d256ce1ee33e1de7e9f3e2cb1a834a0" : [ "file4" ],
        "4cf0ff5673ec65d9900df95502ed92b2605fc602ca20b6901652c7561b302668026095813af6adb0e663bdcdbe1f276d18bf0de254992a78573ad6574e7ae1f6" : [ "file2" ],
        "96a26e7629b55187f9ba3edc4acc940495d582093b8a88cb1f0303cf3399fe6b1f5283d76dfd561fc401a0cdf878c5aad9f2d6e7e2d9ceee678757bb5d95c39e" : [ "file1" ],
        "b10ff867df18165a0e100d99cd3d27f845f7ef9ad84eeb627a53aabaea04805940c3693154b8a32541a31887dda9fb1e667e93307473b1c581021714768bd032" : [ "dir1/file3", "dir1/file4" ]
      }
    }
  }
}